zipline/assets/__init__.py

```diff
@@ -20,6 +20,15 @@
     make_asset_array,
     CACHE_FILE_TEMPLATE
 )
+from .asset_writer import (
+    NullAssetDBWriterLegacy,
+    AssetDBWriterLegacyFromList,
+    AssetDBWriterLegacyFromDictionary,
+    AssetDBWriterLegacyFromDataFrame,
+    AssetDBWriterLegacyFromReadable,
+    FUTURE_TABLE_FIELDS,
+    EQUITY_TABLE_FIELDS
```

Comment:
Why not have the assets.py import them directly from asset_writer.py? That would prevent a circular import.